### PR TITLE
ARGO-596 Stream avro decode - Add HDFS destination

### PR DESCRIPTION
**Pull request by kaggis**
- Include streaming project in compute-engine repo
- Add avro decode step in streaming pipeline
- Add hdfs destination 
- Add job command line parameters for choosing topic, avro_schema and hdfs destination
      `--topic` `--schema` `--hdfs-dest`

### CODE REVIEW

File: streaming/src/main/java/argoflink/ArgoStream.java [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+  public static void main(String[] args) throws Exception  {
+	
+    StreamExecutionEnvironment see = StreamExecutionEnvironment.getExecutionEnvironment();
+    		 
+    ParameterTool parameterTool = ParameterTool.fromArgs(args);
+    
+    try {
+		readSchema(parameterTool.getRequired("schema"));
+	} catch (IOException e) {
+		// TODO Auto-generated catch block
```
Comment (themiszamani) [inline streaming/src/main/java/argoflink/ArgoStream.java]: please print descriptive error messages.

File: streaming/src/main/java/argoflink/ArgoStream.java [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+	}
+	
+	public static GenericRecord avroDec(byte[] payload)  {
+        Decoder decoder = DecoderFactory.get().binaryDecoder(payload, null);
+        GenericRecord payload2;
+		try {
+			payload2 = theReader.read(null, decoder);
+		} catch (IOException e) {
+
```
Comment (themiszamani) [inline streaming/src/main/java/argoflink/ArgoStream.java]: please print descriptive error messages.

File: streaming/src/main/java/argoflink/ArgoStream.java [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+import org.apache.hadoop.hbase.client.HTable;
+import org.apache.hadoop.hbase.client.Put;
+import org.apache.hadoop.hbase.util.Bytes;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonParser;
+
+
+
+public class ArgoStream {
```
Comment (themiszamani) [inline streaming/src/main/java/argoflink/ArgoStream.java]: Please document your code.

**Comment by kaggis** [comment]
please review
/cc @themiszamani

File: streaming/src/main/java/argoflink/ArgoStream.java (hunk 2 above)
Comment (kaggis) [reply streaming/src/main/java/argoflink/ArgoStream.java]: added proper logging

File: streaming/src/main/java/argoflink/ArgoStream.java (hunk 1 above)
Comment (kaggis) [reply streaming/src/main/java/argoflink/ArgoStream.java]: added proper logging

File: streaming/src/main/java/argoflink/ArgoStream.java (hunk 3 above)
Comment (kaggis) [reply streaming/src/main/java/argoflink/ArgoStream.java]: added documentation

**Pull request merged**